### PR TITLE
Change snatch message to 'Started PROPER download' if PROPER

### PR DESCRIPTION
**Pull request by fernandog**
TODO:
Improve proper detection after proper_tags branch gets merged
@p0psicles 

@OmgImAlexis only for PROPER

screenshot_2016-07-05-16-22-56-1

### CODE REVIEW

**Comment by fernandog** [comment]
@duramato can you test this too please ?

File: sickbeard/search.py [hunk 1]
```diff
@@ -103,6 +103,7 @@ def snatchEpisode(result, endStatus=SNATCHED):  # pylint: disable=too-many-branc
         return False
 
     result.priority = 0  # -1 = low, 0 = normal, 1 = high
+    is_proper = False
```
Comment (p0psicles) [inline sickbeard/search.py]: So is_proper is only used, when the sickbeard.NZB_METHOD == 'nzbget'?

File: sickbeard/notifiers/tweet.py [hunk 2]
```diff
@@ -38,9 +38,9 @@ class Notifier(object):
     AUTHORIZATION_URL = 'https://api.twitter.com/oauth/authorize'
     SIGNIN_URL = 'https://api.twitter.com/oauth/authenticate'
 
-    def notify_snatch(self, ep_name):
+    def notify_snatch(self, ep_name, is_proper):
         if sickbeard.TWITTER_NOTIFY_ONSNATCH:
-            self._notifyTwitter(common.notifyStrings[common.NOTIFY_SNATCH] + ': ' + ep_name)
+            self._notifyTwitter(common.notifyStrings[(common.NOTIFY_SNATCH, common.NOTIFY_SNATCH_PROPER)[is_proper]] + ': ' + ep_name)
```
Comment (p0psicles) [inline sickbeard/notifiers/tweet.py]: Better to just use str.format() for this.

File: sickbeard/notifiers/tweet.py [hunk 3]
```diff
@@ -38,29 +38,29 @@ class Notifier(object):
...
         if sickbeard.TWITTER_NOTIFY_ONSUBTITLEDOWNLOAD:
-            self._notifyTwitter(common.notifyStrings[common.NOTIFY_SUBTITLE_DOWNLOAD] + ' ' + ep_name + ": " + lang)
+            self._notifyTwitter('{0} {1}: {2}'.format(common.notifyStrings[common.NOTIFY_SUBTITLE_DOWNLOAD], ep_name, lang))
 
     def notify_git_update(self, new_version="??"):
         if sickbeard.USE_TWITTER:
             update_text = common.notifyStrings[common.NOTIFY_GIT_UPDATE_TEXT]
             title = common.notifyStrings[common.NOTIFY_GIT_UPDATE]
-            self._notifyTwitter(title + " - " + update_text + new_version)
+            self._notifyTwitter('{0} - {1}{2}'.format(title, update_text, new_version)
```
Comment (p0psicles) [inline sickbeard/notifiers/tweet.py]: Do you test before you commit?

File: sickbeard/search.py (hunk 1 above)
Comment (fernandog) [inline sickbeard/search.py]: yes, only nzbget reads the is_proper.

**Comment by p0psicles** [comment]
Approved

**Pull request merged**